lib-utilities/proto/compositionservice/composition_service.pb.go

```diff
@@ -0,0 +1,631 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Headsup: The *pb.go files will be no longer be part of the github repo as these are generated on the go.